### ISSUE TITLE
Handling command failures documentation doesn't state the right method

### ISSUE THREAD

**Issue by JorelAli** [JorelAli/CommandAPI#514]
### Documentation URL

https://commandapi.jorel.dev/9.3.0/commandfailures.html

### What the documentation currently says

```java
throw CommandAPI.failWithString(String message);
throw CommandAPI.failWithMessage(Message message);
throw CommandAPI.failWithBaseComponents(BaseComponent... message);
throw CommandAPI.failWithAdventureComponent(Component message);
```

### Proposed fix

It should state `CommandAPIBukkit.failWithAdventureComponent` (etc.)

**Comment by Xemii16** [comment]
By the way, I noticed it yesterday

**Comment by JorelAli** [comment]
> By the way, I noticed it yesterday

@Xemii16 Yes, thanks very much! I raised this because you mentioned this in the CommandAPI Discord server yesterday so we could keep track of this

**Comment by JorelAli** [comment]
Fixed in version 9.4.0.
